_posts/2020-02-26-warnung_umstellung_11s.md

```diff
@@ -0,0 +1,47 @@
+
+Wie im vorigen Blogpost beschrieben, werden wir damit <!--die im Bremer Freifunk-Netz--> die WLAN-Mesh-Verbindungen zwischen allen Knoten von IBSS auf das neue [802.11s](https://de.wikipedia.org/wiki/IEEE_802.11s)-Verfahren umstellen.
+Die Änderung wird die Unterstützung einiger neuer Routermodelle (wie z.B. einige AVM-Fritzboxen) ermöglichen;
+und sie ist außerdem nötig, weil die [Basis](https://wiki.freifunk.net/Gluon) unserer Bremer Freifunk-Firmware in absehbarer Zeit kein IBSS-Mesh mehr unterstützen wird.
+
+Die ganze Umstellung ist technisch komplex, aber wir haben uns ein Verfahren ausgedacht, welches dafür sorgt, dass die Umstellung problemlos und beinahe unbemerkt über die Bühne geht:  
+am **Stichtag 5. März 2020** werden alle Knoten im Netz gleichzeitig ihr Mesh-Systen von IBSS auf 11s umstellen.
+Zumindest werden das alle Knoten tun, die mit der aktuellen Stable-Firmware [2019.1.1-bremen1](https://wiki.ffhb.de/Firmware/Changelog#freifunk-bremen-versionen_2019-1-1-bremen1) laufen.
+
+**Was ist also zu beachten?**
+- schaltet am besten den **Autoupdater** auf euren Knoten ein (z.B. auf `stable`). Dann wird euer Knoten bis zum 5.3.2020 automatisch das Update installieren und ist für die Umstellung bereit.
+- oder stellt sicher, dass ihr eure Knoten bis zum 5.3.2020 auf die Version **2019.1.1-bremen1** oder neuer aktualisiert habt
+- denkt auch an "eingelagerte" Knoten, die nur gelegentlich angeschaltet werden: nehmt diese vor dem 5.3.2020 kurz (eine Stunde sollte genügen) in Betrieb, damit sie das Update installieren können.
+
+Falls einzelne Knoten eures Mesh-Netzwerk zwischenzeitlich nicht erreichbar sind, kann es sein,
+dass während der [Testphase](/blog/2020/01/23/wechsel-von-ibss-zu-11s.html) bereits Teile des Netzwerkes auf 11s umgestellt wurden.
+Dafür gibt es folgende Lösungen:
+- ihr wartet auf den Stichtag 5.3.2020; dann werden sich die restlichen Knoten ebenfalls auf 11s umstellen.
+- ihr stellt die restlichen Geräte manuell (per Config-Modus oder SSH-Zugang) auf 11s um. Mehr dazu steht unter [2019.1+bremen1.1 als neue Testing](/blog/2020/01/23/wechsel-von-ibss-zu-11s.html).
+- ihr stellt die betroffenen Geräte erstmal wieder auf IBSS zurück, bis am 5.3.2020 dann alle Geräte sowieso auf 11s wechseln.
+    Dieser Weg ist allerdings nicht zu empfehlen und wird nur der Vollständigkeit erwähnt.
+
+Übrigens: falls ein Knoten am 5.3. nicht eingeschaltet ist (oder offline ist), macht das nichts - solange der Knoten schon das Update auf die Version 2019.1.1-bremen1 oder neuer bekommen hat.
+Dann wird er sich beim nächsten Einschalten automatisch von IBSS auf 11s umstellen.
+
+Was könnt ihr tun, wenn ihr nach dem 5.3. trotzdem noch einen Knoten habt, der eine zu alte Firmware hat?
+In diesem Fall könnt ihr den Knoten immer noch per Kabel mit einem anderen Freifunk-Knoten (oder auch direkt mit dem Internet) verbinden, damit er sich das Update installiert.
```

Comment:
Weiß nicht ob das gut ist zu erwähnen.
Da dies in der nächsten BATMAN Umstellung nicht möglich ist

Reply:
Naja, im Moment ist dieser Weg ja durchaus noch möglich. Bei der nächsten Umstellung wird es halt etwas schwieriger, aber wir müssen den Schwierigkeitsgrad ja nicht gleich bei der ersten Umstellung auf "hardcore" stellen :-) sondern können dann beim BATMAN-Upgrade darauf hinweisen, dass ein nachträgliches Upgrade schwierig ist.

Reply:
Joa, schon klar - doch ggf. erinnern sich leute an dieser Lösung (bei diesen Problem/Upgrade) und meckern dann, dass es "garnicht" geht. - daher der Einwand

Reply:
Mir fällt halt nix ein, um das im Text kurz und knackig rüberzubringen :-)